packages/venia-ui/lib/RootComponents/Product/product.gql.js

```diff
@@ -0,0 +1,92 @@
+import { gql } from '@apollo/client';
```

Comment:
Hmm, somewhere else I commented that it was interesting that `gql` was the default export but here it is a named export. One of these places might need to be changed.